hudi-client/hudi-spark-client/src/main/scala/org/apache/hudi/HoodieSparkUtils.scala

```diff
@@ -57,6 +57,8 @@ object HoodieSparkUtils extends SparkAdapterSupport {
 
   def isSpark3_2: Boolean = SPARK_VERSION.startsWith("3.2")
 
+  def gteqSpark3_2: Boolean = SPARK_VERSION > "3.2"
+
   def beforeSpark3_2(): Boolean = {
     if (isSpark2 || isSpark3_0 || isSpark3_1) {
```

Comment:
these 2 are logically interchangeable.. can we consolidate them into 1 ?

Reply:
sure, it should be.

Reply:
sure, it should be.